ui/src/kapacitor/containers/KapacitorPage.js

```diff
@@ -168,6 +169,7 @@ KapacitorPage.propTypes = {
     url: string.isRequired,
     kapacitors: array,
   }),
+  location: shape({}),
```

Comment:
could you specify the definition of this prop's shape? it helps other developers know what contract & data shape they can expect when working within this component.

Reply:
did this.